src/player.cpp

```diff
@@ -3182,7 +3184,7 @@ void Player::doAttacking(uint32_t)
 
 		SchedulerTask* task = createSchedulerTask(std::max<uint32_t>(SCHEDULER_MINTICKS, delay), std::bind(&Game::checkCreatureAttack, &g_game, getID()));
 		if (!classicSpeed) {
-			setNextActionTask(task);
+			setNextActionTask(task, false);
 		} else {
 			g_scheduler.addEvent(task);
```

Comment:
does this introduces any inconsistency?

Reply:
Not sure what you mean by that, I added optional parameter to this method, which by default is previous behaviour